urlchecker/core/urlproc.py

```diff
@@ -167,13 +170,13 @@ def check_urls(self, urls=None, retry_count=1, timeout=5):
 
         # if no urls are found, mention it if required
         if not urls:
-            if self.print_all:
+            if self.print_level == "all":
```

Comment:
I don't think this should be subject to print level - this message should be printed to tell the user there are no urls regardless. I would remove this if statement here and restore to how it was.

Reply:
Unfortunately I disagree, I totally get your point but I think merging all printing into one option with multiple choices is much better than having a flag and printing variable which can be confusing in my opinion :/

Reply:
The user needs to know if there are not any urls, which is different from just not printing results. If you want to control verbosity of non url messages then we can discuss adding a logger (also with levels) but shoving all printing under one variable because it’s easy is not appropriate.

Reply:
You got a point but It is not about easy or hard, the changes are not major. I just have the feeling that we are having a lot of input variables at this point, which is nice but it can be confusing to users.